.jenkins/pytorch/test.sh

```diff
@@ -457,7 +457,8 @@ test_forward_backward_compatibility() {
   python -m venv venv
   # shellcheck disable=SC1091
   . venv/bin/activate
-  pip_install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+  DATE_OF_BASE=$(git show -s --format=%cd --date=short ${BASE_SHA})
+  pip_install --pre "torch<1.13.0.dev${DATE_OF_BASE//-/}" -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

Comment:
Feeling iffy about hardcoding the 1.13.0.dev here --> are there any better ways of specifying a nightly prior to a certain date OR dynamically generating the 1.13.0.dev portion?

Reply:
You do have the current pytorch installed before activative this venv. you can get the head's version to generate the 1.13.0 part.

Reply:
One can read a version from [version.txt](https://github.com/pytorch/pytorch/blob/master/version.txt), can't they?

Reply:
I never know if this is generated/updated automatically in some way during the build process?